packages/manager/modules/dbaas-logs/src/logs/order/logs-order.html

```diff
@@ -92,6 +92,7 @@ <h4 data-translate="logs_order_selection_title" class="mt-4"></h4>
             </oui-select-picker>
         </div>
         <h4 data-translate="logs_order_location_title" class="mt4"></h4>
+        <strong data-ng-bind=":: $ctrl.regionNote"></strong>
```

Comment:
for non US region, you'll create an empty `<strong>` tag

```suggestion
        <strong data-ng-if="$ctrl.region === 'US'" data-translate="dbaas_logs_note_avaibility_us"></strong>
```